AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -85,9 +85,16 @@ struct _ABMeasurementsView: View {
                         measurementsTitle
                             .font(Font.moderate(size: 12))
                         streamNames
-                        if !isCollapsed && measurementsViewModel.showLoadingIndicator {
-                            ProgressView()
-                                .progressViewStyle(CircularProgressViewStyle())
+                        if session.type == .mobile && session.deviceType == .AIRBEAM3 {
```

Comment:
all this should be done in a viewmodel since this change is not layout related. It's a view logic and the viewmodel should be responsible for a view logic, not the view itself. I think this file should not be changed at all in order to implement required changes.